examples/inverse/plot_tf_lcmv.py

```diff
@@ -41,10 +41,11 @@
 # Pick a selection of magnetometer channels. A subset of all channels was used
 # to speed up the example. For a solution based on all MEG channels use
 # meg=True, selection=None and add grad=4000e-13 to the reject dictionary.
+# We could do this with a "picks" argument to Epochs and the LCMV functions,
+# but here we use raw.pick_types() to save memory.
```

Comment:
I think we're okay here -- before/after:

![before](https://cloud.githubusercontent.com/assets/2365790/13675486/e2eb1dd2-e6af-11e5-837d-9cce3d4421e4.png)

![after](https://cloud.githubusercontent.com/assets/2365790/13675491/e5e20fbe-e6af-11e5-98ab-484d0f1d8c03.png)